internal/driver/cli.go

```diff
@@ -61,6 +64,10 @@ func parseFlags(o *plugin.Options) (*source, []string, error) {
 	flagContentions := flag.Bool("contentions", false, "Display number of delays at each region")
 	flagMeanDelay := flag.Bool("mean_delay", false, "Display mean delay at each region")
 	flagTools := flag.String("tools", os.Getenv("PPROF_TOOLS"), "Path for object tool pathnames")
+	// HTTPS options
+	flagCertFile := flag.String("cert", "", "TLS client certificate file")
```

Comment:
I like the brevity of the docs but we should here also clarify that this about the TLS params for profile and symbol fetch.